iib/web/api_v1.py

```diff
@@ -1092,3 +1094,50 @@ def get_nested_bundles(request_id: int) -> flask.Response:
 
     with open(related_bundles_file_path) as f:
         return flask.Response(f.read(), mimetype='application/json')
+
+
+@api_v1.route('/builds/fbc-operations', methods=['POST'])
+@login_required
+def fbc_operations() -> Tuple[flask.Response, int]:
+    """
+    Submit a request to run supported fbc operation on an index image.
+
+    :rtype: flask.Response
+    :raise ValidationError: if required parameters are not supplied
+    """
+    payload: FbcOperationRequestPayload = flask.request.get_json()
+    if not isinstance(payload, dict):
+        raise ValidationError('The input data must be a JSON object')
+
+    request = RequestFbcOperations.from_json(payload)
+    db.session.add(request)
+    db.session.commit()
+    messaging.send_message_for_state_change(request, new_batch_msg=True)
+
+    overwrite_from_index = payload.get('overwrite_from_index', False)
+    # TODO - remove # noqa when worker implementation is done
+    celery_queue = _get_user_queue(serial=overwrite_from_index)  # noqa
+
+    # TODO - update order based on worker implementation
+    args = [
+        request.id,
+        payload['fbc_fragment'],
+        payload['from_index'],
+        payload.get('add_arches'),
+        payload.get('build_tags'),
+        payload.get('add_arches'),
```

Comment:
Minor observation and it could need to be changed anyway 
payload.get('add_arches'), is added twice